### ISSUE TITLE
Full Stripe Functionality for Form Builder

### ISSUE THREAD

**Issue by logivity** [CrocoBlock/suggestions#4759]
Have recently tested stripe functionality in form builder and found it to be very limiting. 

Stripe plugin should be able to have functionality to capture payments later and save cards to charge later (for example for crowdfunding) or for subscription purposes.

**Comment by logivity** [comment]
Great example for this functionality is given in link below.

https://demo.paymentsplugin.com/sell-recurring-services-popup/

**Comment by yxknmyo** [comment]
+1. Excepting payments within the form would bring it up to par with competitors like WPForms, too: https://wpforms.com/addons/stripe-addon/

**Comment by ghost** [comment]
+1. Accepting payments within the form would bring it up to par with competitors like WPForms, too: https://wpforms.com/addons/stripe-addon/

**Comment by zup44** [comment]
+100000 asap!

**Comment by morosmo** [comment]
+100000 asap!

**Comment by Topicimes** [comment]
+1

**Comment by noerpol** [comment]
Yeah, REALLY NEED to be able to charge users on a monthly subscription.

**Comment by drewbclark** [comment]
@MjHead @Crocoblock @girafffee @Floydtm

Can we please get this!! It would be so helpful to be able to turn on any payment method that Stripe allows their customers to use with the Stripe Checkout. It would open the door to be able to offer "Buy Now Pay Later" solutions too! PLEASE!!!

Also being able to use their built in Discount/Promotion Code System like shown in @ofmarconi image example would be VERY USEFUL. I can't currently figure out a way to do this with the current version of Stripe Integration with JetForm Builder.

Also being able to map Jet Form Builder Form Data to map it into the metadata on Stripe and link user data as a Customer on Stripe. Great to keep orders organized and be able to look back and find details quicker.

I also wish I could name the "Product Name" on checkout instead of it just using the "Blog/Site Title" like: My Site payment.

Just better integration with Stripe is much needed!

Really need these changes for it to feel like a PRO feature. I love Crocoblock and everything you guys bring and this would absolutely make it so much better having these features.
